pkg/monitor/monitor.go

```diff
@@ -235,7 +235,7 @@ func (m *Monitor) cleanupDriverNodePods(node *volume.NodeInfo) {
 		if err != nil {
 			return false, nil
 		}
-		if node.Status == volume.NodeOffline || node.Status == volume.NodeDegraded {
+		if n.Status == volume.NodeOffline || n.Status == volume.NodeDegraded {
```

Comment:
Can we check why existing UTs isn't hitting this line?

Reply:
There is no UT or an integration test exercising a node which goes Offline but comes back Online within the monitor timeout. The current UT framework just fakes data, so I would go for an integration test to exercise this scenario. cc @Rohit-PX